README.md

```diff
@@ -68,7 +72,7 @@ TODO: Link a dedicated roadmap.
 ## Support
 If you find a bug please file an [issue](https://github.com/scylladb/scylla-operator/issues) for us.
 
-We are also available on `#scylla-operator` channel on [Slack](https://scylladb-users-slackin.herokuapp.com/) if you have questions.
+We are also available on `#scylla-operator` channel on [Slack](https://scylladb-users.slack.com) if you have questions.
 
 ## Contributing
 We would **love** you to contribute to Scylla Operator, help make it even better and learn together! Have a look at the [Contributing Guide](docs/source/contributing.md) or reach out to us on `#scylla-operator` channel on [Slack](https://scylladb-users-slackin.herokuapp.com/) if you have questions.
```

Comment:
Link to Slack channel in "Contributing" section is still invalid. Please update it from https://scylladb-users-slackin.herokuapp.com/ to https://scylladb-users.slack.com

Reply:
Ah - it appears twice. I fixed it in line 75, did not notice line 78.